cv/pothole_detection/src/pothole_detection_model_inference.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+import json
+import struct
+
+import cv2
+from cv2 import intersectConvexConvex
+import numpy as np
+import onnx
+import onnxruntime as ort
+import redis
+import rospkg
+from runonnx import run_inference
+
+
+class CVModelInferencer:
+    def __init__(self):
+        self.redis = redis.Redis(host='127.0.0.1', port=6379, db=3)
+        
+        rospack = rospkg.RosPack()
+        model_path = f"{rospack.get_path('pothole_detection')}/models/best.onnx"
```

Comment:
We may need to have a different model name for this

Also we don't have the model commit here so just remember that as a note

Reply:
bad practice to upload large files to git. We have them stored on google drive